app/views/spam2/_comments.html.erb

```diff
@@ -71,4 +71,10 @@ $(document).ready(function () {
       </tbody>
     </table>                       
   </div>
-</div>
\ No newline at end of file
+</div>
+<div class="page-table">
```

Comment:
You haven't closed this div :)

Reply:
Could you format this HTML as well (indent it)?

Reply:
Sure, done Thanks!!

Reply:
Great catch :tada: